app/src/main/java/toasty/messageinabottle/io/LiveBackend.java

```diff
@@ -54,4 +58,39 @@
             return result;
         }
     }
+
+    public static boolean tryToCreateAccount(String username, String password) throws IOException {
+        RequestBody formBody=new FormBody.Builder()
```

Comment:
I don't think that the API endpoint supports forms for the body. I'm pretty sure that we need to send a json object in the body instead. We might have to ask Nik.

Creating the json object shouldn't be too hard though:
e.g.
```java
Map<String, String> myMap = new HashMap<String, String>();
myMap.put("one", "hello");
myMap.put("two", "world");
String json = gson.toJson(myMap);
```